### PR TITLE
Remove static import

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Remove ``import static rtypes``
This PR unifies all the examples.

# Testing this PR

Check that example job is green

# Related reading

See https://trello.com/c/9nZfJ6Pi/132-import-static-cleanup
and https://github.com/openmicroscopy/openmicroscopy/pull/4758

### CODE REVIEW

**Comment by joshmoore** [comment]
I had forgotten about the tests. Especially for them, I don't see the benefit. But in general, I might now be lightly pro-`import static`. :smile: I'll leave you all to discuss while I'm on vacation though.

**Comment by jburel** [comment]
For the tests, the major changes are the legacy usage of JUnit: very few rtypes stuff.
This PR unifies all the examples and removes some import not used at all.

**Comment by mtbc** [comment]
Certainly :+1: for not mixing testing frameworks. I don't mind writing `Assert.assertEquals` and suchlike in tests, no strong opinions here, so happy to be blown by other winds.

File: examples/OmeroClients/updates.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import static omero.rtypes.*;
+
```
Comment (mtbc) [inline examples/OmeroClients/updates.java]: few double-blank lines being introduced

**Comment by dominikl** [comment]
Apart from the double empty lines, looks alright 👍

**Comment by jburel** [comment]
@dominikl: done

**Comment by dominikl** [comment]
Thanks, good to merge 👍

**Pull request merged**